src/power_divergence.jl

```diff
@@ -387,6 +387,11 @@ function ChisqTest(x::AbstractVector{T}, y::AbstractVector{T}, k::T) where T<:In
     PowerDivergenceTest(d, lambda=1.0)
 end
 
+function ChisqTest(x::AbstractVector{T}, y::AbstractVector{T}) where {T<:Integer}
```

Comment:
Please include documentation explaining how to use this method!

Reply:
I added the documentation for this new method.